code/Readme.md

```diff
@@ -1 +1,3 @@
 This folder contains all source code and test code.
+
+Test for push.
```

Comment:
Don't forget to remove this line.
And btw, please check our [Conventional Commit rules](https://www.conventionalcommits.org/en/v1.0.0/) to refine the commit message. @andyasdd1